Security/TwoFactor/AuthenticationContextInterface.php

```diff
@@ -63,4 +63,11 @@ public function isAuthenticated();
      * @param bool $authenticated
      */
     public function setAuthenticated($authenticated);
+
+    /**
+     * Is authentication request.
+     *
+     * @return bool
+     */
+    public function isAuthenticationTry();
```

Comment:
@scheb but is it actually an extension point and a public interface? Is it implied by this bundle to be ever extended by the final user?